src/Layer/C3DTilesLayer.js

```diff
@@ -386,21 +386,14 @@ class C3DTilesLayer extends GeometryLayer {
                     if (c3DTileFeature.object3d != object3d) {
                         continue;// this feature do not belong to object3d
                     }
+
+                    this._style.context.setFeature(c3DTileFeature);
+
```

Comment:
The `Style` class provides a `setContext` method to pass a custom `Context`. You could then provide your custom context with a single parameter `properties` to not break the old style function declaration. You could update here this property for each `feature`.

Reply:
I did not set a custom context but just the geometry of the style context because the setter was already there.

Reply:
Okay for me, but we'll surely rework this with a custom context on the new 3D Tiles implementation. =)